app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -88,10 +88,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     mSwipesButton.setOnClickListener(this);
     mBrbButton.setOnClickListener(this);
 
+    // Resetting the buttons on page reload
+    changeButtonColor(R.color.blue, R.color.wash, mNorthButton);
```

Comment:
Take a look at `WeeklyMenuFragment`. We might want to make a list of buttons or button references and loop through for the button logic.